src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php

```diff
@@ -77,7 +77,7 @@ public function getFunctions()
         ];
     }
 
-    public function load($uuid)
+    public function load($uuid, bool $includeExtensions = true, array $includedProperties = null)
```

Comment:
another possibility would be a `array $properties` parameter that matches the semantic of the `properties` param of the `page_selection`

Reply:
I would prefer that the `sulu_content_load` would work the same way as `page_selection` currently not sure if the `page_selection` outputs any `view` data. I think it would be good if it would do that. Still I think the mapping configuration with `excerptImages => excerpt.images` is a nice feature of the page_selection and make it easier for twig developer to access this things.

Reply:
done 🙂